### ISSUE TITLE
Staging Review finding: Focus is not being managed on "Continue"

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#127613]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** BIO - Pingwind
**Product:** VA Form 21-8940 Veteran's Application for Increased Compensation Based on Unemployability
**Feature:** Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

From "Let's confirm VA Form 21-8940 is the right form for your needs" to "Please read this important information before you start:", the focus is not managed and does not move to the heading on the new page, which may be confusing for screen reader and other assistive technology users.

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="304" height="819" alt="focus management" src="https://github.com/user-attachments/assets/57b7430d-263f-4e74-9fac-094e9c278e52" />

</details>

## Recommended action

Ensure that focus is managed appropriately between screens, to provide context for all users. 

https://design.va.gov/accessibility/focus-management#when-a-new-page-loads

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 2.4.3 Focus Order - Level A](https://www.w3.org/TR/WCAG22/#focus-order)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
